### ISSUE TITLE
authorities of 2 synonyms

### ISSUE THREAD

**Issue by sjmeades** [Canadensys/vascan-data#1761]
[Originally posted on GoogleCode (id 1739) on 2013-04-09]

<b>(This is the template to report a data issue for Vascan. If you want to</b>
<b>report another issue, please change the template above.)</b>

<b>What is the URL of the page where the problem occurs?</b>
http://data.canadensys.net/vascan/taxon/8735

<b>What data are incorrect or missing?</b>
Dryas integrifolia subsp. chamissonis (Sprengel) H.J. Scoggan 
Dryas chamissonis Juzepczuk

<b>What data are you expecting instead?</b>
Dryas integrifolia subsp. chamissonis (Sprengel ex Juzepczuk) Scoggan 
?Dryas chamissonis Sprengel ex Juzepczuk

<b>If applicable, please provide an authoritative source.</b>
IPNI authors for Scoggan

There is a confusing duplication of Dryas chamissonis entries in IPNI.  One set has Spreng.ex Juz (1929), the other has just Juz.  The 1929 publication is listed as a nom. nudum. in IPNI
For the 1984 name by Juzupczuk, IPNI states in one entry that this is a homonym.  
Tropicos accepts the 1929 D.chamissonis Spreng. ex Juz., but does't list the 1984 name.  Yet they leave out the basionym in the combination D. integrifolia subsp. chamissonis Scoggan.
IPNI list both D. integrifolia subsp. chamissonis (Juz.) Scoggan
and D. integrifolia subsp. chamissonis (Spreng.) Scoggan.
so, you may need to check with Kanchi Gandhi about this one.
None of them seem correct.

The Plant List uses D. chamissonis Spreng. ex Juz.

**Comment by brouille** [comment]
[Originally posted on GoogleCode on 2013-04-10 20:33Z]

needs consultation

**Comment by brouille** [comment]
[Originally posted on GoogleCode on 2013-06-12 20:04Z]

Answer from K. Gandhi:
"Dryas chamissonis Spreng. ex Juzepczuk, Izv. Glavn. Bot. Sada S.S.S.R. 28: 312, 326-327. 1929: Valid
We did not have the 1929 publ.
Tom Zanoni (NYBG) sent me the relevant pages.  I saw the name as validly published (with a scanty description on p. 327; everyone in the past missed this page).
But there was a question from my discussion group whether Juzepczuk accepted the name at the rank of species (or not).  I did not want to take a chance and provided all the info Tom sent me.
Werner confirmed its validity.

Note: "Dryas chamissonis Jurtzev, Arktichesk. Fl. SSSR 9(1): 322, 1984" is a later homonym (NOT an isonym).
Kanchi"

Therefore: Dryas chamissonis Spreng. ex Juzepczuk
and Dryas chamissonis Jurtzev nom. illeg. - I am not adding this one since I am uncertain as to whether it applies to the same taxon.

Luc
